Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/box.yml

```
@@ -272,6 +272,8 @@
       orGroup: Pizza
     - id: FoodPizzaCotton
       prob: 0.10
+    - id: FoodPizzaWorldpeas
+      prob: 0.05
       orGroup: Pizza
```

Looks like you accidentally stole the `orGroup` line from the cotton pizza. That's causing the box to over fill, which is why the test is failing.
```suggestion
     - id: FoodPizzaCotton
       prob: 0.10
       orGroup: Pizza
    - id: FoodPizzaWorldpeas
      prob: 0.05
      orGroup: Pizza
```